<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmRegister</class>
 <widget class="QWidget" name="FrmRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrierung</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Registrierung</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Benutzername:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtUsername">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>FireDiver</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Passwort:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPassword1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>12345678</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Passwort wiederholen:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPassword2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>12345678</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtEmail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>firediver98@gmail.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Best√§tigungscode:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtCode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Code an Email senden</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
